index.html

```diff
@@ -9,31 +9,458 @@
-    <article>
+        </nav>
+    </header>
+    <section id="landing">
         <h1><span>Hey there, I’m "damdafayton".
             </span><span> I’m a software developer.</span></h1>
-        <section>I can help you build a product , feature or website Look through some of my work and experience! If
+        <p id="landing-p">I can help you build a product , feature or website Look through some of my work and
+            experience! If
             you
             like what you see and have a project you need coded, don’t hestiate to contact me.
-        </section>
-        <div id="contact-box">
-            <a href="https://github.com/backtofayton"><img src="./icons/github.png" alt=""></a>
-            <a href="https://linkedin.com/in/damdafayton"><img src="./icons/linkedin.png" alt=""></a>
-            <a href="https://twitter.com/topragagiden"><img src="./icons/twitter.png" alt=""></a>
+        </p>
+        <ul id="contact-box">
+            <li><a href="https://github.com/backtofayton"><img src="./icons/github.png" alt=""></a></li>
+            <li><a href="https://linkedin.com/in/damdafayton"><img src="./icons/linkedin.png" alt=""></a></li>
+            <li><a href="https://twitter.com/topragagiden"><img src="./icons/twitter.png" alt=""></a></li>
+        </ul>
+        <svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <rect x="1" y="1" width="40" height="40" rx="20" fill="white" />
+            <path fill-rule="evenodd" clip-rule="evenodd"
+                d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z"
+                fill="#172B4D" />
+            <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6" />
+        </svg>
+    </section>
+    <section>
+        <div>
+            <h1>My Recent Works</h1>
+            <svg width="49" height="4" viewBox="0 0 49 4" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect x="0.5" width="48" height="4" rx="2" fill="#36B37E" />
+            </svg>
+        </div>
+        <div class="card-container">
+            <div class="card">
+                <img src="./works/work1.jpg" alt="">
+                <h1>Uber</h1>
+                <div class="tech">
+                    <div class="tech-tag">HTML</div>
+                    <div class="tech-tag">CSS</div>
+                </div>
+                <button>See Project</button>
+            </div>
+            <div class="card">
+                <img src="./works/work2.jpg" alt="">
+                <h1>Facebook</h1>
+                <div class="tech">
+                    <div class="tech-tag">HTML</div>
+                    <div class="tech-tag">CSS</div>
+                </div>
```

Comment:
- [ ] Consider wrapping the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") in a `ul` and each of them a `li`. This is because the list of technologies in each project card are unordered lists and therefore should be in a `ul` and `li` tag. See suggestion below:

```suggestion
                <ul class="tech">
                    <li class="tech-tag">HTML</li>
                    <li 

- [ ] 

- [ ] 

- [ ] class="tech-tag">CSS</li>

                </ul>
```